service/routes.py

```diff
@@ -207,6 +207,33 @@ def get_item(order_id, item_id):
     return make_response(jsonify(message), status.HTTP_200_OK)
 
 
+######################################################################
+# DELETE AN ITEM FROM AN ORDER
+######################################################################
+@app.route("/orders/<int:order_id>/items/<int:item_id>", methods=["DELETE"])
+def delete_item(order_id, item_id):
+    """
+    Delete an item from an order
+    """
+    app.logger.info("Request to delete an Item %s from Order with id: %s", item_id, order_id)
+    
+    # See if the order exists and abort if it doesn't
+    order = Order.find(order_id)
+    if not order:
+        abort(
+            status.HTTP_404_NOT_FOUND,
+            f"Order with id '{order_id}' could not be found.",
+        )
```

Comment:
As discussed with professor, there is no need to check whether the order exists in delete case. The API will return HTTP_204_NO_CONTENT in all cases.